project_dev_docs/phase1_resources/interview_script.md

```diff
@@ -0,0 +1,44 @@
+# Interview Script for User Research into Open Practices
+
+**Running time:** 30-45 mins
+
+## Context
+
+_What do we want to find out?_
+
+- Interview different people across different themes and at different levels (for example, researcher, project manager, project lead, and so on)
+- Assess the current attitudes towards open working at the Turing
+- Uncover barriers and hidden fears towards open working
```

Comment:
Add hypotheses on what these are. Series of "I think" statements. The data will then turn these into "we know" statements, or uncover something different.